pegasus/test/fixtures/fake_dashboard.rb

```diff
@@ -73,6 +73,29 @@ module FakeDashboard
   ]
 
   #
+  # Fake Data: Courses
+  #
+  COURSES = [
+    {
+      id: 15,
+      name: 'csp',
+      created_at: '2016-01-01 00:01:02',
+      updated_at: '2016-01-01 00:01:02'
+    }
+  ]
+
+  #
+  # Fake Data: Scripts
+  #
+  SCRIPTS = [
+    {
+      id: 10,
+      name: 'flappy',
+      hidden: 0
+    }
+  ]
```

Comment:
Though it is great to see the `scripts` table being seeded, I'm not seeing usage of this in the tests. Is that intentional?